client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js

```diff
@@ -45,6 +45,10 @@ function LaptopForm() {
 			.then((data) => {
 				setRequestId(data.id);
 				setNavigate(true);
+				const welcomeNotify = "We have received you request ";
+				Notification.requestPermission().then((prem) => {
+					const notification = new Notification(`${welcomeNotify}`);
```

Comment:
(?) Probably we don't want to actually send a notification here, only request permission

Reply:
(?) And actually, let's move the request permission to the laptop-request-status page, at the same place as we register to receive socket events (in case during demo we give people access to a status page to see the live update)

Reply:
done, I removed the notification request from the laptop request page and kept it only on the laptop-request-status page.